package.json

```diff
@@ -85,11 +85,11 @@
     "@babel/preset-react": "^7.8.3",
     "@babel/register": "^7.4.4",
     "@storybook/addon-a11y": "^5.3.14",
```

Comment:
For some reason, the latest version get's stuck at "Initialising", and I can't find any related issues. For now I'll leave it here, and we can upgrade separately later.